src/v3/src/components/InformationalText/InformationalText.tsx

```diff
@@ -28,6 +28,7 @@ const InformationalText: UISchemaElementComponent<{
       justifyContent="flex-start"
     >
       <Typography
+        id={uischema.id}
```

Comment:
I'm a little bit concerned that users will start to rely on these `id`s to hook into the DOM. Is this `id` dynamic (as in has some randomness to it) or are they deterministic?